Add backslashes to hyphens to get real minus signs with groff.

diff -ru perl-5.8.8.orig/lib/Net/Config.pm perl-5.8.8/lib/Net/Config.pm
--- perl-5.8.8.orig/lib/Net/Config.pm	2004-07-04 00:47:13.000000000 +0200
+++ perl-5.8.8/lib/Net/Config.pm	2004-07-04 01:04:55.000000000 +0200
@@ -157,7 +157,7 @@
 Attempts to determine if a given host is outside your firewall. Possible
 return values are.
 
-  -1  Cannot lookup hostname
+  \-1  Cannot lookup hostname
    0  Host is inside firewall (or there is no ftp_firewall entry)
    1  Host is outside the firewall
 
diff -ru perl-5.8.8.orig/lib/Net/FTP.pm perl-5.8.8/lib/Net/FTP.pm
--- perl-5.8.8.orig/lib/Net/FTP.pm	2004-07-04 00:47:13.000000000 +0200
+++ perl-5.8.8/lib/Net/FTP.pm	2004-07-04 01:04:55.000000000 +0200
@@ -1243,7 +1243,7 @@
     $ftp = Net::FTP->new("some.host.name", Debug => 0)
       or die "Cannot connect to some.host.name: $@";
 
-    $ftp->login("anonymous",'-anonymous@')
+    $ftp->login("anonymous",'\-anonymous@')
       or die "Cannot login ", $ftp->message;
 
     $ftp->cwd("/pub")
diff -ru perl-5.8.8.orig/lib/Net/SMTP.pm perl-5.8.8/lib/Net/SMTP.pm
--- perl-5.8.8.orig/lib/Net/SMTP.pm	2004-07-04 00:47:13.000000000 +0200
+++ perl-5.8.8/lib/Net/SMTP.pm	2004-07-04 01:04:55.000000000 +0200
@@ -558,7 +558,7 @@
 
 This example prints the mail domain name of the SMTP server known as mailhost:
 
-    #!/usr/local/bin/perl -w
+    #!/usr/local/bin/perl \-w
 
     use Net::SMTP;
 
@@ -569,7 +569,7 @@
 This example sends a small message to the postmaster at the SMTP server
 known as mailhost:
 
-    #!/usr/local/bin/perl -w
+    #!/usr/local/bin/perl \-w
 
     use Net::SMTP;
 
